client/src/app/seller/_containers/seller-all-review-container.tsx

```diff
@@ -0,0 +1,67 @@
+'use client'
+
+import { Icon } from '@iconify/react/dist/iconify.js'
+import { useEffect, useState } from 'react'
+import { toast } from 'sonner'
+
+import { Text } from '@/components/text'
+
+import type { ArrayResponseReview } from '@/generated/api'
+import { getReview } from '@/generated/api'
+
+import { SellerReviewCard } from '../_components/seller-review-card'
+
+type SellerAllReviewContainerProps = {
+  sellerId: string
+}
+
+export function SellerAllReviewContainer({
+  sellerId,
+}: SellerAllReviewContainerProps) {
+  const [reviews, setReviews] = useState<ArrayResponseReview>({
```

Comment:
มี type ที่ไม่ใช่ Response มั้ย ไม่อยากให้ใช้ `x.data.y`

```suggestion
  const [reviews, setReviews] = useState<ArrayResponseReview>({
```